src/features/foods/components/FilterBadge.tsx

```diff
@@ -0,0 +1,44 @@
+import { CrossIcon } from '@/assets/images/icons';
+import { Icon } from '@/components/ui';
+import { Badge } from '@/components/ui/Badge';
+import { cn } from '@/lib/tailwind/utils';
+
+import { ComponentType } from 'react';
+
+interface IFilterBadgeProps {
+  icon?: ComponentType;
+  emoji?: string;
+  text: string;
+  className?: string;
+  onCrossClick?: () => void;
+}
+
+const FilterBadge = ({
+  icon,
+  emoji,
+  text,
+  className,
+  onCrossClick,
+  ...props
+}: IFilterBadgeProps) => {
+  const CircledIcon = icon;
+  return (
+    <Badge variant="light" className={cn('pl-1 pr-0 gap-0', className)} {...props}>
```

Comment:
This is more like a question, (sorry).
What if we want to change the color of the Badge in FilterBadge?
For now, filterbatch will be used with one color, so it seems okay now. but I am  curious a little.

Reply:
We could add a new variant to the parent `Badge` component if the color of the `FilterBadge` is completely renewed. Contrastly, suppose a new style is *added* to the `FilterBadge` (meaning multiple variants are used for the `FilterBadge`). In that case, we may need to add a new variant to `FilterBadge` and may also have to separate it from the parent `Badge` component as needed.